### DISCUSSION TITLE
Issue Launching magick (ImageMagick-7.1.1-Q16-HDRI)

### DISCUSSION THREAD

**Discussion by nrf5033** [ImageMagick/ImageMagick#7503] [General] [answered]
Hi - Beginner ImageMagick user here. magick keeps crashing each time I try to launch it - does anyone have any ways to fix / ways around this? 

OS: Windows 10

Background:

--------------

About a month ago, I was able to use magick v7.1.1 to create a spritesheet for myself using magick montage (see screenshots attached of my successful usage). Coming back to the program now, I'm attempting to launch it and run the command prompt I had done before (using the latest version 'ImageMagick-7.1.1-Q16-HDRI'), but now each time I attempt to open magick, the command window appears for a split second and then disappears. Not sure if this is a crash? I've uninstalled / reinstalled v7.1.1 (and have also tried the older v6.9.13-13-Q16-HDRI-x64-dll separately), and I keep encountering this issue. There's likely something obvious I'm missing, but any help is greatly appreciated.

![spritesheet](https://github.com/user-attachments/assets/3e00fcc1-a23e-499c-910d-c5d73073bcd0)
![Successful Magick Terminal Command June 2024](https://github.com/user-attachments/assets/53096c0a-7d2a-44a7-868f-a7b1abb40990)

**Comment by snibgo** [comment]
> I'm attempting to launch it and run the command prompt ...  the command window appears for a split second and then disappears.

The usual method to use `magick` in Windows is to first open a command window. This is a window that is typically white text on a black background, and shows a prompt like `C:\Users\Alan>` and waits for you to type or paste your command and hit "enter". Then it will execute your command. It won't open another command window. If there was a problem, the shell or IM will show an error message. Then the shell displays another prompt and waits for your next command.

Other methods to run `magick` include scripts such as Windows BAT or Windows Powershell or bash. These also will not open another command window.

**Reply by nrf5033** [reply, marked as answer]
I figured it out - I had been launching Command Prompt from windows start. I needed to type 'cmd' inside windows file explorer in the folder where the images were located. Once there, I had the right access / syntax to simply execute montage.
